### ISSUE TITLE
Tool Not working in Arc 10.7 and 10.7.1 for TomTom processing

### ISSUE THREAD

**Issue by Tya30** [Esri/street-data-processing-tools#34]
I found on 3 different PCs having all ArcMap 10.7 or 10.7.1 that the toolbox was not running correctly. Access to "Unblock" (Security) needed to uncheck in the installation was possible on 1 of 3 PCs.
When I ran the "Processes MultiNet Street Data" tool (with TomTom dataset), only the computer that installed correctly worked but stopped running (after 2:30 am) reporting this problem:

_**Exception from HRESULT: 0x800422B8
   at ESRI.ArcGIS.Geodatabase.IDatasetContainer2.CreateDataset (IDEDataset DataElement)
   Has GPProcessVendorDataFunctions.ProcessMultiNetDataFunction.CreateAndBuildNetworkDataset (String outputFileGdbPath Double fgdbVersion, fdsName String, String ndsName, createNetworkAttributesInMetric Boolean, Boolean createTwoDistanceAttributes, timeZoneIDBaseFieldName String, Boolean directedTimeZoneIDFields, commonTimeZone String, Boolean usesHistoricalTraffic, ITrafficFeedLocation trafficFeedLocation, usesRSTable Boolean, Boolean usesLTRTable, usesLRSTable Boolean, String lrsStatsTableName)
   at GPProcessVendorDataFunctions.ProcessMultiNetDataFunction.Execute (IArray paramvalues, ITrackCancel trackcancel, IGPEnvironmentManager envMgr, IGPMessages messages)
Failed to execute (ProcessMultiNetData).**_

I succeeded in ArcMap 10.6.1. My problem is solved.

Hoping to help you improve this toolbox.

**Comment by mmorang** [comment]
I apologize that we never responded to this problem, but I'm glad switching to ArcMap 10.6.1 resolved your problem back in 2019 when you logged the issue.

The ArcMap version of the tool was rebuilt at some point for the final release of ArcMap (10.8.*), and ArcMap is now in mature support.  A new version of the Street Data Processing Tools for ArcGIS Pro is now available, and this version of the toolbox does not require installation or have any dlls that have to be unblocked.

Closing this issue as no longer relevant.
